custom/project/VirtuaFeaturedProducts/Resources/config.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
plik niepotrzebny na razie

Reply:
A w ogóle, to masz zdublowany plugin - widzę go zarówno w custom/plugins i custom/project. Powinien być tylko w custom/project